LibTest/io/Link/createSync_A01_t01.dart

```diff
@@ -3,34 +3,33 @@
 // BSD-style license that can be found in the LICENSE file.
 
 /// @assertion void createSync(
-///  String target, {
-///  bool recursive: false
-///  })
+/// String target,
+/// {bool recursive = false}
```

Comment:
Is it actually intended to have no indentation?:
```suggestion
///   String target,
///   {bool recursive = false}
```